### ISSUE TITLE
Cannot copy object. The source object 'default' does not exist

### ISSUE THREAD

**Issue by Technolink** [aws/aws-sdk-net#591]
In the PowerShell SDK we are seeing the following problem:
```PowerShell
Set-AWSCredentials -AccessKey a -SecretKey b -StoreAs default
Initialize-AWSDefaults -Region us-east-1 -ProfileName default
```

In AWSPowerShell module version 3.3.57.0 this works. In version 3.3.64.1 (latest) it gives the following error:
```PowerShell
Initialize-AWSDefaults : Cannot copy object. The source object 'default' does not exist.
At line:1 char:1
+ Initialize-AWSDefaults -Region us-east-1 -ProfileName default
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Initialize-AWSDefaults], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException,Amazon.PowerShell.Common.InitializeDefaultsCmdlet
```

This is causing a lot of issues in my company (we have filed a separate ticket with AWS support).

**Comment by vellozzi** [comment]
We've reproduced the issue and are working on a fix now.

**Comment by Technolink** [comment]
@vellozzi Thanks so much! Do you have even a very rough ETA to get a new msi out? I have a workaround (use a profile name besides 'default'), so I'm debating waiting for your fix or doing the workaround.

**Comment by vellozzi** [comment]
It won't be until late today at the earliest.  If you can use the workaround that would be great.

**Comment by vellozzi** [comment]
The fix for this has been released in AWS Tools for PowerShell 3.3.64.2.
